docs/site/site.xml

```diff
@@ -41,6 +41,7 @@
         <menu name="Related Articles">
             <item name="DZone" href="https://dzone.com/articles/how-to-transform-any-type-of-java-bean-with-one-li" />
             <item name="InfoQ" href="https://www.infoq.com/articles/expedia-rid-of-bean-transformers/" />
+            <item name="InfoQ Brazil" href="https://www.infoq.com/br/articles/expedia-rid-of-bean-transformers/" />
```

Comment:
if you don't mind, I would leave the link only in the readme file

Reply:
I was testing it, and Brazilian is a little bit strange, because we speak Brazilian Portuguese, so I think it's going to be better with Portuguese, like this:
<item name="InfoQ" collapse="true" href="index.html">
                <item name="English" href="https://www.infoq.com/articles/expedia-rid-of-bean-transformers/" />
                <item name="Portuguese" href="https://www.infoq.com/articles/expedia-rid-of-bean-transformers/" />
            </item>

Reply:
fine with me